mungegithub/github/github.go

```
@@ -398,6 +401,14 @@ func (config *Config) PreExecute() error {
 		Transport: transport,
 	}
 	config.client = github.NewClient(client)
+	rawurl := config.Url
+	if len(rawurl) > 0 {
+		url, err := url.Parse(rawurl)
+		if err != nil {
+			glog.Fatalf("Unable to parse url: %v: %v", rawurl, err)
```

Thanks for failing early